contrib/pmsnare/pmsnare.c

```diff
@@ -177,31 +197,45 @@ CODESTARTparse
 			--lenMsg;
 			++p2parse;
 		}
-		dbgprintf("pmsnare: separator [%d]'%s'  msg after the timestamp and hostname: [%d]'%s'\n", tablength,
-		TabRepresentation,lenMsg, p2parse);
-		if(lenMsg > 13 && strncasecmp((char*) p2parse, "MSWinEventLog", 13) == 0) {
-			snaremessage=13; /* 0 means not a snare message, a number is how long the tag is */
+		dbgprintf("pmsnare: separator [%d]'%s'  msg after the timestamp and "
+				"hostname: [%d]'%s'\n", 
+				tablength,TabRepresentation,lenMsg, p2parse);
+		if(lenMsg > 13 && strncasecmp( \
+				(char*) p2parse, "MSWinEventLog", 13) == 0) {
+			len_snare_tag=13;
 		}
-		if(lenMsg > 11 && strncasecmp((char*) p2parse, "LinuxKAudit", 11) == 0) {
-			snaremessage=11; /* 0 means not a snare message, a number is how long the tag is */
+		if(lenMsg > 11 && strncasecmp( \
+				(char*) p2parse, "LinuxKAudit", 11) == 0) {
```

Comment:
I saw from recent commit messages that you had started enforcing a line length limit, but I thought that it was 80 characters instead of 120. I'll remove the wrapping.